config/routes.rb

```diff
@@ -66,6 +66,7 @@
   devise_scope :user do
     get '/users/signup' => 'devise/registrations#new', :as => :new_user_registration
     post '/users' => 'devise/registrations#create', :as => :user_registration
+    get "sign_out", to: "devise/sessions#destroy"
```

Comment:
Adding a route to sign out with an HTTP GET request (previously, you had to send an HTTP DELETE request)